tests/rkt_auth_test.go

```diff
@@ -0,0 +1,289 @@
+// Copyright 2015 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/coreos/rkt/Godeps/_workspace/src/github.com/ThomasRooney/gexpect"
+	taas "github.com/coreos/rkt/Godeps/_workspace/src/github.com/endocode/test-aci-auth-server/lib"
+	"github.com/coreos/rkt/rkt/config"
+)
+
+func TestAuthSanity(t *testing.T) {
+	skipUnsafe(t)
+	removeDataDir(t)
+	server := runServer(t, taas.None)
+	defer server.Close()
+	runRktBang(t, server.URL, "none1")
+}
+
+func TestAuthBasic(t *testing.T) {
+	skipUnsafe(t)
+	removeDataDir(t)
+	defer removeAllConfig(t)
+	server := runServer(t, taas.Basic)
+	defer server.Close()
+	basicNoConfig(t, server)
+	basicWithCustomConfig(t, server)
+	basicWithVendorConfig(t, server)
```

Comment:
Those 3 functions are only called 1 time. Is there any reason not to inline them here (in a different paragraph)? I find it easier to read the code linearly rather than jumping from functions to functions. Same question for other tests.